src/Database/Console/Migrations/FreshCommand.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Hyn\Tenancy\Database\Console\Migrations;
+
+use Hyn\Tenancy\Traits\MutatesMigrationCommands;
+use Illuminate\Database\Console\Migrations\FreshCommand as BaseCommand;
+class FreshCommand extends BaseCommand
+{
+    use MutatesMigrationCommands;
+
+    /**
+     * Execute the console command
+     */
+    public function handle()
+    {
+        if (!$this->confirmToProcess()) {
```

Comment:
According to the unit tests this method does not exist. Have you ran the tests locally with success?

See https://circleci.com/gh/hyn/multi-tenant/1666

Reply:
Will look into it! I think I am missing something small here.